cmd/oras/internal/display/status/progress/manager.go

```diff
@@ -92,17 +92,17 @@ func (m *manager) render() {
 	defer m.statusLock.RUnlock()
 	// todo: update size in another routine
 	width, height := m.console.Size()
-	len := len(m.status) * 2
+	statusLength := len(m.status) * 2
```

Comment:
How about `lineCount`, one status output take two lines.
```suggestion
	lineCount := len(m.status) * 2
```

Reply:
Done